src/util.jl

```diff
@@ -8,13 +8,26 @@ for f in [:to_one_based_indexing, :to_zero_based_indexing]
     end
 end
 
-get_current_app() = shell_execute("print \$User::application->name;")[2]
+get_current_app() = shell_execute("print \$User::application->name;").stdout
 
 function get_docs(input::String; full::Bool=true, html::Bool=false)
     pos = UInt(max(length(input)-1, 0))
     return shell_context_help(input, pos, full, html)
 end
 
+function shell_execute(str::AbstractString)
+    flag, out, err, msg = _shell_execute(str)
+    correct_input = Bool(flag)
+    if correct_input
+        isempty(msg) && return (stdout=out, stderr=err)
+        throw(PolymakeError(msg))
```

Comment:
It might make sense to have some way to see stdout and stderr in the throw case as well as there can be some useful output. Maybe with a `@show` and/or a verbose flag?